src/ContaoCommunityAlliance/DcGeneral/Contao/Dca/Populator/BackendViewPopulator.php

```diff
@@ -159,6 +160,11 @@ protected function populatePanel(EnvironmentInterface $environment)
                 }
             }
         }
+
+        $baseConfig    = $environment->getBaseConfigRegistry()->getBaseConfig();
+        $listingConfig = $viewDefinition->getListingConfig();
+
+        ViewHelpers::initializeSorting($panel, $baseConfig, $listingConfig);
```

Comment:
Sounds reasonable but don't put the priority too high. The default action listeners are priority 0 AFAIR and therefore a priority of 10 should be sufficient.